hyphae/src/main.rs

```diff
@@ -70,19 +59,30 @@ fn sync_blocks(kubo: &KuboApi, myceli: &MyceliApi) -> Result<()> {
                 continue;
             }
         };
-        if let Err(e) = kubo.put_block(&cid, &block) {
-            error!("Error sending block {cid} to kubo: {e}");
-            all_blocks_synced = false;
+        match kubo.put_block(&cid, &block) {
+            Err(e) => {
+                error!("Error sending block {cid} to kubo: {e}");
+                all_blocks_synced = false;
+            }
+            Ok(resp) => {
+                if cid == resp.key {
+                    info!("Synchronized {}", &cid);
+                } else {
+                    error!("Synchronizing {} actually resulted in {}", &cid, resp.key);
+                }
+                kubo_blocks.insert(cid, block.clone());
+                sleep(Duration::from_millis(500));
```

Comment:
Why sleep?

Reply:
Paranoia. hyphae is a non-time-sensitive background program that I don't want to ever consume lots of resources all at once.

That said, removed this.